src/components/footer/index.tsx

```diff
@@ -67,8 +63,17 @@ function Footer({
 			>
 				<InlineSvg className={s.logo} src={svgHashicorpLogo} />
 			</a>
+			<ButtonLink
+				text="Give Feedback"
+				href={FEEDBACK_FORM_URL}
+				color="secondary"
+				size="small"
+				icon={<IconPencilTool16 />}
+				opensInNewTab={true}
+				className={s.feedbackButton}
+			/>
```

Comment:
🚧 In the viewport widths where this element visually appears last in the footer, we need to make sure that this element is focused last. Currently, it's always the second focusable element in the `<footer>`. I believe this falls under [SC 2.4.3: Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html).

![image](https://user-images.githubusercontent.com/43934258/210662054-31dcb6b3-2a69-4037-867d-8456bec649bf.png)

We should be able to render this element twice and then use CSS to only show one of them at a time based on the visual order of the elements.

Reply:
Ah I didn't realize. thank you!

Reply:
@ashleemboyer just pushed up the updates based on your notes 😄

Reply:
Taking another look now! 👀